packages/code-editor/src/CodeEditor/languages/sql.ts

```diff
@@ -0,0 +1,238 @@
+          label: tableName,
+          kind: monaco.languages.CompletionItemKind.Variable,
+          insertText: tableName,
+          documentation: `Table name: ${tableName}`,
+          sortText: "0", // Prioritize table names on the suggestions list
+        }));
+
+        // Return both table suggestions and keyword suggestions
+        return { suggestions: [...tableSuggestions, ...keywordSuggestions] };
+      }
+
+      if (
+        (isLastWordKeyword &&
+          showColumnsKeywords.some((k) =>
+            k.endsWith(lastWord.toUpperCase()),
+          )) ||
+        (!isLastWordKeyword &&
+          showColumnsKeywords.some((k) =>
+            k.endsWith(secondLastWord.toUpperCase()),
+          )) ||
+        lastWord === ""
+      ) {
+        const columnsSuggestions = columnNames.map((columnName) => ({
+          label: columnName,
+          kind: monaco.languages.CompletionItemKind.Variable,
+          insertText: columnName,
+          documentation: `Column name: ${columnName}`,
+          sortText: "0", // Prioritize column names on the suggestions list
+        }));
+
+        // Return both table suggestions and keyword suggestions
+        return { suggestions: [...columnsSuggestions, ...keywordSuggestions] };
+      }
+
+      return { suggestions: keywordSuggestions };
+    },
+  };
+};
+
+/**
+ * Validates SQL and creates error markers to show on code editor.
+ */
+export const hvSqlValidator = async (
+  content: string,
+  editor: any,
+  monaco: Monaco,
+) => {
+  const model = editor.getModel();
+  try {
+    format(content);
+    return [];
+  } catch (error) {
+    const errorObj = parseErrorMessage((error as Error).message);
+
+    return [
+      {
+        severity: monaco.MarkerSeverity.Error,
+        message: errorObj?.message ?? "",
+        startLineNumber: errorObj?.line ?? 1,
+        startColumn: errorObj?.column ?? 1,
+        endLineNumber: errorObj?.line ?? 1,
+        endColumn: model.getLineLastNonWhitespaceColumn(errorObj?.line),
+      },
+    ];
+  }
+};
+
+/** SQL custom options. */
+export const hvSqlOptions = {};
+
+/**
+ * SQL code formatter.
+ * When the code has errors, it is not formatted and the orginal code is returned.
+ * @param content Current code editor content
+ * @returns `string with the formatted code or `undefined`
+ *
+ * SQL Formatter options and demo:
+ * https://www.npmjs.com/package/sql-formatter
+ * https://sql-formatter-org.github.io/sql-formatter/
+ */
+export const hvSqlFormatter = async (
+  content: string,
+  options?: FormatOptionsWithLanguage,
+) => {
+  try {
+    const formattedContent = format(content, {
+      tabWidth: 2,
+      indentStyle: "tabularLeft",
+      ...options,
+    });
+    // Format only if there are no errors
+    return formattedContent;
+  } catch (error) {
+    return content;
+  }
+};
+
+export interface SqlLanguagePlugin extends Omit<LanguagePlugin, "schema"> {}
+
+export const sqlLanguagePlugin: SqlLanguagePlugin = {
```

Comment:
we don't have `"schema"` anymore
```suggestion
export const sqlLanguagePlugin: LanguagePlugin = {
```